src/lib/y2storage/proposal/autoinst_drives_map.rb

```diff
@@ -46,13 +46,16 @@ class AutoinstDrivesMap
       #   @see #each_pair
       def_delegators :@drives, :each, :each_pair
 
+      # CT_MD is not supported yet
+      SUPPORTED_TYPES = [:CT_DISK, :CT_LVM].freeze
+
       # Constructor
       #
       # @param devicegraph  [Devicegraph] Devicegraph where the disks are contained
       # @param partitioning [Array<Hash>] Partitioning layout from an AutoYaST profile
       def initialize(devicegraph, partitioning)
         # By now, consider only regular disks
-        disks = partitioning.select { |i| i.fetch("type", :CT_DISK) == :CT_DISK }
+        disks = partitioning.select { |i| SUPPORTED_TYPES.include?(i.fetch("type", :CT_DISK)) }
```

Comment:
Hmmm, is it possible to have a LVM_CT without `<device>`? If so, it will be matched to nothing (only disks are considered in `#first_usable_disk`). If not... that's impossible to spot from this code with apparently makes no difference between disks and VGs.